tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -164,24 +160,117 @@ public void processOsm(SourceFeature sf, FeatureCollector features) {
 
       if (sf.hasTag("highway", "motorway_link", "trunk_link", "primary_link", "secondary_link",
         "tertiary_link")) {
-        feat.setAttr("link", 1);
+        feat.setAttr("is_link", true);
       }
 
       // Set "brunnel" (bridge / tunnel) property where "level" = 1 is a bridge, 0 is ground level, and -1 is a tunnel
       // Because of MapLibre performance and draw order limitations, generally the boolean is sufficent
-      // See also: "layer" for more complicated ±6 layering for more sophisticated graphics libraries
       if (sf.hasTag("bridge") && !sf.hasTag("bridge", "no")) {
-        feat.setAttrWithMinzoom("level", 1, 12);
+        feat.setAttrWithMinzoom("is_bridge", true, 12);
       } else if (sf.hasTag("tunnel") && !sf.hasTag("tunnel", "no")) {
-        feat.setAttrWithMinzoom("level", -1, 12);
-      } else {
-        feat.setAttrWithMinzoom("level", 0, 12);
+        feat.setAttrWithMinzoom("is_tunnel", true, 12);
       }
 
       // Server sort features so client label collisions are pre-sorted
       feat.setSortKey(minZoom);
 
       OsmNames.setOsmNames(feat, sf, minZoomNames);
+    } // end highway=
+
+    // non-highway features
+    // todo: exclude railway stations, levels
```

Comment:
This is important, there were several Tilezen versions where we didn't have that filtering and the rendering was borked.